art/art_param.py

```diff
@@ -757,7 +786,19 @@
             "fancy71": [fancy71_dic, False],
             "fancy72": [fancy72_dic, False],
             "fancy73": [fancy73_dic, False],
-            "fancy74": [fancy74_dic, False]
+            "fancy74": [fancy74_dic, False],
+            "fancy75": [fancy75_dic, True],
+            "fancy76": [fancy76_dic, False],
+            "fancy77": [fancy77_dic, True],
```

Comment:
Why you add some fonts with `True` flag and complete dictionary?
This item is for memory saving, when a font is not case sensitive and you add it with `True` flag you should remove uppercase characters from it's dictionary, but it's completely **optional**.